packages/utils/__tests__/types.vitest.ts

```diff
@@ -0,0 +1,81 @@
+import * as vue from 'vue'
+import * as vueShared from '@vue/shared'
+import * as vueuse from '@vueuse/core'
+import { describe, expect, it } from 'vitest'
+import {
+  isArray,
+  isBoolean,
+  isDate,
+  isElement,
+  isEmpty,
+  isFunction,
+  isNumber,
+  isObject,
+  isPromise,
+  isPropAbsent,
+  isString,
+  isSymbol,
+  isUndefined,
+  isVNode,
+} from '..'
+
+describe('types', () => {
+  it('re-export from @vue/shared', () => {
+    expect(isArray).toBe(vueShared.isArray)
+    expect(isDate).toBe(vueShared.isDate)
+    expect(isFunction).toBe(vueShared.isFunction)
+    expect(isObject).toBe(vueShared.isObject)
+    expect(isPromise).toBe(vueShared.isPromise)
+    expect(isString).toBe(vueShared.isString)
+    expect(isSymbol).toBe(vueShared.isSymbol)
+  })
+
+  it('re-export from vueuse', () => {
+    expect(isBoolean).toBe(vueuse.isBoolean)
+    expect(isNumber).toBe(vueuse.isNumber)
+  })
+
+  it('re-export from vue', () => {
+    expect(isVNode).toBe(vue.isVNode)
+  })
+
+  it('isUndefined should work', () => {
+    expect(isUndefined(undefined)).toBe(true)
+    expect(isUndefined(null)).toBe(false)
+    expect(isUndefined('null')).toBe(false)
```

Comment:
I don't think that we should test the correctness of methods in 3rd party libraries, since we assume that they are correct.